packages/block-library/src/comments-pagination/edit.js

```diff
@@ -73,14 +76,28 @@ export default function QueryPaginationEdit( {
 		<>
 			{ hasNextPreviousBlocks && (
 				<InspectorControls>
-					<PanelBody title={ __( 'Settings' ) }>
-						<CommentsPaginationArrowControls
-							value={ paginationArrow }
-							onChange={ ( value ) => {
-								setAttributes( { paginationArrow: value } );
-							} }
-						/>
-					</PanelBody>
+					<ToolsPanel
+						label={ __( 'Settings' ) }
+						resetAll={ () =>
+							setAttributes( { paginationArrow: 'none' } )
+						}
+					>
+						<ToolsPanelItem
+							label={ __( 'Arrow Controls' ) }
+							hasValue={ () => paginationArrow !== undefined }
```

Comment:
```suggestion
							hasValue={ () => paginationArrow !== 'none' }
```

The hasValue prop should only be true if the value has been changed from its default value. Since the default value for an arrow is none, we should check that value.

Reply:
- [x] Update the hasValue function check